README.md

```diff
@@ -64,3 +62,10 @@ As a Maker
 So that I can find relevant peeps
 I want to filter on a specific keyword
 ```
+
+## Screenshots
+* Chitter homepage
+![alt text](https://github.com/CristinaGonzalezVaro/chitter-challenge-apprenticeships/blob/main/Chitter%20homepage.png)
+
+* Posting new peep
+![alt text](https://github.com/CristinaGonzalezVaro/chitter-challenge-apprenticeships/blob/main/Chitter%20new%20peep.png)
```

Comment:
This seems like a really useful idea to include in your README!

Reply:
Agreed! The screenshots are a really nice touch